### PR TITLE
Animated wallpaper project(new project)  please verify and merge

### PR DESCRIPTION
**Pull request by harshsinghcs**
this an animation wallpaper i  made

## 🛠️ Issue (Number)

Closes #

# Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] I have made this from my own
- [ ] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] The title of my pull request is a short description of the requested changes.

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
###Screenshot

Screenshot (64)

Screenshot (59)

### CODE REVIEW

**Comment by harshsinghcs** [comment]
I have made the Pull request please approve it @zugzwang03  @TusharKesarwani  @harshit-paneri

**Comment by harshsinghcs** [comment]
> Issue number?

https://github.com/TusharKesarwani/Front-End-Projects/issues/1842

File: Projects/Animated-Wallpaper/colourfull.png
Comment (zugzwang03) [inline Projects/Animated-Wallpaper/colourfull.png:1]: Where did you use this image?

File: Projects/Animated-Wallpaper/oranges.jpg
Comment (zugzwang03) [inline Projects/Animated-Wallpaper/oranges.jpg:1]: Where did you use this image?

**Review by zugzwang03 — CHANGES_REQUESTED** [review]

**Comment by zugzwang03** [comment]
Give the user options to change text and background, otherwise its use case reduces to huge extent

**Comment by harshsinghcs** [comment]
> Give the user options to change text and background, otherwise its use case reduces to huge extent

now user have the option to change the text, color and image too

**Comment by harshsinghcs** [comment]
now kindly review my pr and marge it🙏

**Review by zugzwang03 — APPROVED** [review]

**Review by TusharKesarwani — APPROVED** [review]

**Pull request merged**